### ISSUE TITLE
Stop the unstable releases from showing up in Windows package manager (winget)

### ISSUE THREAD

**Issue by astromasoud** [Eugeny/tabby#8130]
Platform: Windows 11 Pro x64
Tabby Version: all

For those who don't know, Windows OS has a package manager sorta like apt on Ubuntu, it's called "winget"!

You can check the updates for the installed programs via running `winget upgrade` (--include-unknown)

![image](https://user-images.githubusercontent.com/18737721/226828056-ee86aabf-1daa-4eb9-a65a-50af913b691d.png)

>> update=upgrade, same.

And then you run `winget upgrade --all` (--include-unknown) to upgrade the programs listed to their latest versions.

The problem is that Tabby's pre- and unstable releases all show up there as stable releases and would be installed automatically if you update all the apps. For example yesterday it installed version [v1.0.194](https://github.com/Eugeny/tabby/releases/tag/v1.0.194) of Tabby for me which turned out to be unusable and unstable and I had to download the latest stable release from the website and downgrade it to version [Alpha 189](https://github.com/Eugeny/tabby/releases/tag/v1.0.189).

Also today I noticed that a new stable version is available on Github, version [Alpha 196](https://github.com/Eugeny/tabby/releases/tag/v1.0.196), but as you see in the image above it's not showing in the `winget upgrade` results. Maybe it shows up later as Microsoft's repos update or maybe it has something to do with the naming.

Long story short, please look into this and prevent the unstable and pre- releases from being installed via winget. You should probably add some tag to define those versions as not-latest so that Microsoft won't pick them up as being stable releases!
